app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,10 @@ html lang="#{I18n.locale}" class='no-js'
       meta http-equiv="refresh" content="#{yield(:meta_refresh)}"
 
     title
-      = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
```

Comment:
the `+` ends up creating an HTML-unsafe value, where this approach will check each piece separate for HTML-safeness before rendering (the result of the `yield` may already be HTML safe, as it was in the buggy case)